Pipfile

```diff
@@ -17,6 +17,7 @@ cython = "==0.29.*"
 dask = "*"
 dill = "==0.3.*"
 distributed = "*"
+elyra-python-editor-extension = "*"
```

Comment:
We need to run `jupyter lab build`, this plugin extension requires that command to work as it is a server extension, at least until Elyra team migrate to jupyterlab3 compatible installation.

Reply:
the assemble script executes the `jupyter lab build`, https://github.com/thoth-station/s2i-generic-data-science-notebook/blob/ed7004160d96921ba9d63afef8035fb90676ef9b/.s2i/bin/assemble#L14

however the jupyter lab build seems to fail. Need to work on it.